HCore-Web/Startup/Startup.cs

```diff
@@ -366,6 +370,13 @@ protected virtual void ConfigureGenericServices(IServiceCollection services)
             services.AddScoped<IUrlProvider, UrlProviderImpl>();
             services.AddScoped<INonHttpContextUrlProvider, NonHttpContextUrlProviderImpl>();
             services.AddScoped<INowProvider, NowProviderImpl>();
+
+            services.AddScoped((serviceProvider) =>
```

Comment:
If (useSpa) notwendig?

Reply:
Hier nicht, weil nur der Cache-Header konfiguriert wird und in jedem Fall `app.UseStaticFiles(staticFileOptions);`  ausgeführt wird. Da können wir es wegnehmen.

Reply:
Das UseSpa hat nix dezidiert mit den Static Files zu tun (siehe oben). Es indiziert dass eine SPA verwendet wird.